static/styles/rewards/gift-cards.css

```diff
@@ -1,193 +1,178 @@
+  justify-content: center;
 }
-.redeem-info-wrapper {
-  position: absolute;
-  z-index: 1000;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background-color: var(--background-color-default);
+.card-section .pricing .available div .currency div {
+  font-size: 8px;
+}
+.card-section .pricing .available div .amount {
+  font-size: 24px;
+  color: #fff;
 }
 .redeem-info {
-  position: fixed;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  max-width: 563px;
-  padding: 24px;
-  border-collapse: collapse;
-  border-color: gray;
-  overflow-y: auto;
-  z-index: 1;
+  border-color: var(--light-mode-border-color);
 }
-
-.close-btn {
-  position: absolute;
-  top: 0;
-  right: 0;
-  cursor: pointer;
-  color: #fff;
+.redeem-info p {
+  text-transform: none;
+}
+.redeem-info fieldset legend {
+  text-align: center;
+  padding: 0 4px;
+}
+.redeem-info fieldset .instructions {
+  padding: 12px;
 }
 
-.claim-gift-card-btn[data-loading="true"] > span.action {
+.btn[data-loading="true"] > .icon {
   display: none;
 }
-.claim-gift-card-btn[data-loading="true"] > span.loading {
-  display: flex;
+.btn[data-loading="true"] > .action {
+  display: none;
 }
-.claim-gift-card-btn[data-loading="true"] > span.loading svg {
-  animation: rotate 1s linear infinite;
+.btn[data-loading="true"] > .loader {
+  display: block;
 }
-.claim-gift-card-btn[data-loading="false"] > span.action {
-  display: flex;
+.btn[data-loading="true"] > .loader svg {
+  animation: rotate 1s linear infinite;
 }
-.claim-gift-card-btn[data-loading="false"] > span.loading {
-  display: none;
+.btn[data-loading="false"] > .icon {
+  display: block;
 }
-.gift-cards-wrapper.purchased .claim-gift-card-btn {
+.btn[data-loading="false"]:hover > .icon {
   display: none;
 }
-
-#reveal-btn[data-loading-reveal="false"] .loading {
+.btn[data-loading="false"] > .action {
   display: none;
 }
-#reveal-btn[data-loading-reveal="false"] .action {
+.btn[data-loading="false"]:hover > .action {
   display: block;
 }
-
-#reveal-btn[data-loading-reveal="true"] .loading {
-  display: block;
+.btn[data-loading="false"] > .loader {
+  display: none;
 }
-#reveal-btn[data-loading-reveal="true"] .loading svg {
-  animation: rotate 1s linear infinite;
+
+.btn {
+  width: 100%;
+  border: none;
+  color: #808080;
+  cursor: pointer;
+  height: 48px;
+  background: 0 0;
 }
 
-#reveal-btn[data-loading-reveal="true"] .action {
-  display: none;
+.btn:hover {
+  cursor: pointer;
+  background-color: #80808020;
+  color: #fff;
 }
+
 .disclaimer {
   text-align: center;
-  padding-top: 24px;
+  padding: 24px 12px;
 }
 
 @media (prefers-color-scheme: light) {
-  .gift-card,
-  .redeem-info-wrapper {
+  .card-section {
     background-color: #fff;
+    box-shadow: inset 0 0 96px #00000008;
   }
-  .gift-card h3,
-  .gift-card .pricing .available,
-  .gift-card div.buttons button,
-  .close-btn {
+
+  .card-section img {
+    border-color: var(--light-mode-border-color);
```

Comment:
Same with this variable

Reply:
This comes under `@media (prefers-color-scheme: light) {`. I tried removing it and the border color for light theme wasn't changing even after rebuild. It was still the same as the dark theme. So, I am keeping it for now.

Reply:
I remember making a build script to dynamically generate the light mode styles based on the dark mode styles. 

Basically only dark mode styles should be authored and then the rest is generated. So I think that using this light mode variable might be wrong.

Reply:
Ok, I am going to take a look into this again.

Reply:
I have looked into it. I see that similar CSS is specified in this repository on some other places too. For example this 
https://github.com/ubiquity/pay.ubq.fi/blob/1ae72801d41394a0843d7f9797abcf9f9a5de89d/static/styles/rewards/light-mode.css#L78
and this 
https://github.com/ubiquity/pay.ubq.fi/blob/1ae72801d41394a0843d7f9797abcf9f9a5de89d/static/styles/rewards/light-mode.css#L64

So I think there is no such script that automatically writes color for different themes in this repository. 
Maybe you are referring to the invert style script from the repository `work.ubq.fi` [ invert-colors.ts](https://github.com/ubiquity/work.ubq.fi/blob/development/build/plugins/invert-colors.ts)

I have also looked into the history of the build script of this repository and I couldn't find a custom script being used in this repository for generating different CSS based on themes.

Reply:
Oh interesting. Perhaps we should implement it. It should cut styling work in half.

Reply:
Ok, I will add a new issue for this as this will require refactoring the existing css to work with the script. The script can probably be copied from `work.ubq.fi` but it will still require debugging. Also, this repository doesn't bundle CSS to one file, I will add that to the new issue.

Reply:
I have added a new issue. https://github.com/ubiquity/pay.ubq.fi/issues/292